frontend/app/controllers/sign-in.js

```diff
@@ -0,0 +1,68 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+/*---- Login Screen ---*/
+
+  loginFailed: false,
+  loginFailedMessage: "Failed to Login",
+  isProcessing: false,
+  isSlowConnection: false,
+  timeout: null,
+
+    success: function(response) {
```

Comment:
Ember js comes with babel. Which means, you can use upto ES6 javascript in your code. So these functions can be written as 

```js
success(response)  {

}
```

Instead of 

```js
success: function(response) {

}
```